### ISSUE TITLE
export fatjar

### ISSUE THREAD

**Issue by quintesse** [jbangdev/jbang#1501]
It would be nice to have a "fatjar" export option.

**Comment by maxandersen** [comment]
It's nontrivial. Requires processing every jar/class and merge files in meta-inf. 

Last I looked none of existing plugins for maven/gradle looked reusable for it.

**Comment by quintesse** [comment]
Perhaps, but we don't need to start with a perfect solution. If we just unpack the deps into the same folder as the app and jar it all up it will work for a lot of cases already.

**Comment by maxandersen** [comment]
You will be surprised how little will work though due to META-INF collisions.

**Comment by quintesse** [comment]
> You will be surprised how little will work though due to META-INF collisions.

Well, I implemented this anyway. Just a stupidly simple implementation.
And it runs all of my scripts without any problems.
I also tried a simple Quarkus one and even that one worked.

So I'm sure there will be lots of examples where it won't work, but for those that do it's pretty awesome to be able to do this,

I did make it print out a message saying it's an experimental feature and it might not work for some applications. With a link to our github issues page.
